tests/pipeline/nodes/dabble/test_tracking.py

```diff
@@ -0,0 +1,76 @@
+"""
+Copyright 2021 AI Singapore
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import os
+import pytest
+import numpy as np
+from peekingduck.pipeline.nodes.dabble.tracking import Node
+
+
+@pytest.fixture
+def size():
+    return (400, 600, 3)
```

Comment:
Is this fixture necessary? can a constant `SIZE = (400, 600, 3)` be used instead?

Reply:
Updated to use constant instead.

Reply:
Using a constant `SIZE = (400, 600, 3)` results in "fixture 'SIZE' not found" error.
Reverting back to initial fixture as per:
https://github.com/aimakerspace/PeekingDuck/blob/dev/tests/pipeline/nodes/dabble/test_bbox_to_btm_midpoint.py#L24

Reply:
Tests still fail with recommended changes. Will keep current implementation.